Monika After Story/game/script-topics.rpy

```diff
@@ -16165,3 +16165,40 @@ label monika_giving_criticism:
     m 3dsu "Graciously accept you can't change everyone's mind and stay considerate when assessing someone else's work."
     m 3hub "...That'd be Monika's Critique Tip of the Day, ahaha!"
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_curse_words",
+            category=["advice", "life"],
+            prompt="Curse words",
+            random=True,
+            pool=False
+        )
+    )
+
+label monika_curse_words:
+    m 3etd "Say, {w=0.2}[player], {w=0.2}do you use curse words sometimes?"
+    m 1hksdlb "Ahaha, {w=0,2}I mean, {w=0,2}it's alright if you swear occasionally, {w=0.2}and I don't have anything against it if you really do."
```

Comment:
```suggestion
    m 1hksdlb "Ahaha, I mean, it's alright if you swear occasionally, a lot of people do after all."
```

Reply:
Honestly, I'm not sure where exactly I should put pauses. They seem pretty appropriate mostly, but I'm really in doubt about this case.